setup/www/tasks/site-setup.yaml

```diff
@@ -41,6 +42,7 @@
     - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh nodejs'
     - '*  *    * * *   nodejs  /home/nodejs/check-build-site.sh iojs'
     - '*  *    * * *   root    /home/nodejs/cdn-purge.sh'
+    - '* */4   * * *   nodejs  rsync -avz --delete benchmark:charts/ /home/www/benchmarking/charts/'
```

Comment:
Suggesting you remove `-v` -- no need to be verbose. Perhaps even replace with `-q` unless you want cron output sent to your mail.